src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexBoyerMoore.cs

```diff
@@ -334,37 +334,35 @@ public int Scan(string text, int index, int beglimit, int endlimit)
         }
 
 #if DEBUG
-        /// <summary>
-        /// Used when dumping for debugging.
-        /// </summary>
+        /// <summary>Used when dumping for debugging.</summary>
         [ExcludeFromCodeCoverage]
         public override string ToString() => Pattern;
 
         [ExcludeFromCodeCoverage]
         public string Dump(string indent)
         {
-            StringBuilder sb = new StringBuilder();
+            var sb = new StringBuilder();
 
             sb.AppendLine($"{indent}BM Pattern: {Pattern}");
-            sb.Append(indent + "Positive: ");
-            for (int i = 0; i < Positive.Length; i++)
+
+            sb.Append($"{indent}Positive: ");
+            foreach (int i in Positive)
             {
-                sb.Append(Positive[i].ToString(CultureInfo.InvariantCulture) + " ");
+                sb.Append(i);
```

Comment:
You are losing the space between the numbers here. Is that intended? For example, how can you tell the difference between `1` and `3` vs `13`?

Reply:
Nope, not intended.